### DISCUSSION TITLE
Determine overlaps with OBO

### DISCUSSION THREAD

**Discussion by cmungall** [CommonCoreOntology/CommonCoreOntologies#395] [General]
1. Figure out how much overlap there is between CCO and OBO (either OBO-Core or specific domain ontologies)
 2. Come up with a strategy: equivalence axioms vs ceding by one side

 * [ ] CORE: only matches I see are document & plan
 * [ ] ENVO: https://github.com/EnvironmentOntology/envo/issues/625

**Comment by cmungall** [comment]
following are automated and include obvious false positives

## IAO

|IAO|IAO label|CCO|CCO label|
|---|---|---|---|
|IAO:0000013|journal article|CCO:JournalArticle|Journal Article|
|IAO:0000030|information content entity|CCO:InformationContentEntity|Information Content Entity|
|IAO:0000064|algorithm|CCO:Algorithm|Algorithm|
|IAO:0000088|report|CCO:Report|Report|
|IAO:0000101|image|CCO:Image|Image|
|IAO:0000112|example of usage|CCO:example_of_usage|example of usage|
|IAO:0000115|definition|CCO:definition|definition|
|IAO:0000117|term editor|CCO:term_editor|term editor|
|IAO:0000119|definition source|CCO:definition_source|definition source|
|IAO:0000132|lot number|CCO:LotNumber|Lot Number|
|IAO:0000136|is about|CCO:ModalRelationOntology/is_about|is about|
|IAO:0000136|is about|CCO:is_about|is about|
|IAO:0000310|document|CCO:Document|Document|
|IAO:0000600|elucidation|CCO:elucidation|elucidation|

## OBI

|OBI|OBI label|CCO|CCO Label|
|---|---|---|---|
|OBI:0000245|organization|CCO:Organization|Organization|
|OBI:0000260|plan|CCO:PlanSpecification|Plan|
|OBI:0000753|Waters|CHEBI:15377|water|
|OBI:0000844|hospital|CCO:Hospital|Hospital|
|OBI:0000940|optical microscope|CCO:OpticalMicroscope|Optical Microscope|
|OBI:0000967|container|CCO:Container|Container|
|OBI:0000990|electron microscope|CCO:ElectronMicroscope|Electron Microscope|
|OBI:0400022|Cyan|CCO:Cyan|Cyan|
|OBI:0400064|laser|CCO:Laser|Laser|
|OBI:0400107|computer|CCO:Computer|Computer|
|OBI:0400169|microscope|CCO:Microscope|Microscope|

## PATO

|CCO|CCO Label|OBO|OBO label|
|---|---|---|---|
|CCO:Acceleration|Acceleration|PATO:0001028|acceleration|
|CCO:Albedo|Albedo|PATO:0001295|albedo|
|CCO:Amount|Amount|PATO:0000070|amount|
|CCO:Amplitude|Amplitude|PATO:0000080|amplitude|
|CCO:AngularVelocity|Angular Velocity|PATO:0001413|angular velocity|
|CCO:Bent|Bent|PATO:0000617|bent|
|CCO:BiologicalSex|Biological Sex|PATO:0000047|biological sex|
|CCO:Black|Black|PATO:0000317|black|
|CCO:Blond|Blond|PATO:0001264|light yellow|
|CCO:Blue|Blue|PATO:0000318|blue|
|CCO:Blunt|Blunt|PATO:0001950|blunt|
|CCO:Bow|Bow|PATO:0000406|curved|
|CCO:Branched|Branched|PATO:0000402|branched|
|CCO:Brown|Brown|PATO:0000952|brown|
|CCO:Circumference|Circumference|PATO:0001648|circumference|
|CCO:Closure|Closure|PATO:0000136|closure|
|CCO:Coiled|Coiled|PATO:0000404|coiled|
|CCO:Color|Color|PATO:0000014|color|
|CCO:ColorBrightness|Color Brightness|PATO:0000016|color brightness|
|CCO:ColorHue|Color Hue|PATO:0000015|color hue|
|CCO:ColorSaturation|Color Saturation|PATO:0000017|color saturation|
|CCO:Combustion|Combustion|PATO:0015021|combustibility|
|CCO:Cuboidal|Cuboidal|PATO:0001872|cuboid|
|CCO:Cyan|Cyan|PATO:0000319|cyan|
|CCO:Cylindrical|Cylindrical|PATO:0001873|cylindrical|
|CCO:Density|Mass Density|PATO:0001019|mass density|
|CCO:Depth|Depth|PATO:0001595|depth|
|CCO:Diameter|Diameter|PATO:0001334|diameter|
|CCO:Drooping|Drooping|PATO:0002165|drooping|
|CCO:Fatigability|Fatigability|PATO:0001815|fatigability|
|CCO:Flat|Flat|PATO:0000407|flat|
|CCO:Fluorescence|Fluorescence|PATO:0000018|fluorescence|
|CCO:Folded|Folded|PATO:0001910|folded|
|CCO:Force|Force|PATO:0001035|force|
|CCO:Frequency|Frequency|PATO:0000044|frequency|
|CCO:Green|Green|PATO:0000320|green|
|CCO:Grey|Grey|PATO:0000950|grey|
|CCO:Hardness|Hardness|PATO:0000048|hardness|
|CCO:Height|Height|PATO:0000119|height|
|CCO:HighFrequency|High Frequency|PATO:0000380|increased frequency|
|CCO:Initialism|Initialism|PATO:0002325|onset quality|
|CCO:Length|Length|PATO:0000122|length|
|CCO:Loudness|Loudness|PATO:0001521|sound amplitude|
|CCO:LowFrequency|Low Frequency|PATO:0000381|decreased frequency|
|CCO:Magenta|Magenta|PATO:0000321|magenta|
|CCO:Magnetic|Magnetism|PATO:0001682|magnetism|
|CCO:Maroon|Maroon|PATO:0001426|maroon|
|CCO:Mass|Mass|PATO:0000125|mass|
|CCO:ModalRelationOntology/asWKT|as WKT|http://www.opengis.net/ont/geosparql#asWKT|as WKT|
|CCO:ModalRelationOntology/has_part|has_part|BFO:0000051|has_part|
|CCO:ModalRelationOntology/part_of|part_of|BFO:0000050|part_of|
|CCO:ModalRelationOntology/precedes|precedes|PATO:0001632|anterior to|
|CCO:Momentum|Momentum|PATO:0001023|momentum|
|CCO:Oblong|Oblong|PATO:0000946|oblong|
|CCO:Opacity|Opacity|PATO:0000957|opacity|
|CCO:Opaque|Opaque|PATO:0000963|opaque|
|CCO:Orange|Orange|PATO:0000953|orange|
|CCO:Organization|Organization|PATO:0000938|organized|
|CCO:Perimeter|Perimeter|PATO:0001711|perimeter|
|CCO:Phosphorescence|Phosphorescence|PATO:0001298|phosphorescence|
|CCO:Power|Power|PATO:0001024|power|
|CCO:Pressure|Pressure|PATO:0001025|pressure|
|CCO:Protruding|Protruding|PATO:0001598|protruding|
|CCO:Purple|Purple|PATO:0000951|purple|
|CCO:Pyramidal|Pyramidal|PATO:0002336|pyramidal|
|CCO:Radioactive|Radioactive|PATO:0001740|activity (of a radionuclide)|
|CCO:Radiopacity|Radiopacity|PATO:0002136|radiopacity|
|CCO:Radiopaque|Radiopaque|PATO:0002137|radiopaque|
|CCO:Rectangular|Rectangular|PATO:0001867|rectangular|
|CCO:Red|Red|PATO:0000322|red|
|CCO:Refractivity|Refractivity|PATO:0001372|refractivity|
|CCO:Rosy|Rosy|PATO:0001425|rosy|
|CCO:Round|Round|PATO:0000411|circular|
|CCO:Scar|Scar|PATO:0001850|scarred|
|CCO:Semicircular|Semicircular|PATO:0002232|semicircular|
|CCO:Serrated|Serrated|PATO:0001206|serrated|
|CCO:Sharp|Sharp|PATO:0000944|sharpness|
|CCO:Sloped|Sloped|PATO:0001481|sloped|
|CCO:SoundFrequency|Sound Frequency|PATO:0001520|sound frequency|
|CCO:SoundWavelength|Sound Wavelength|PATO:0001523|sound wavelength|
|CCO:Speed|Speed|PATO:0000008|speed|
|CCO:Spherical|Spherical|PATO:0001499|spherical|
|CCO:Split|Split|PATO:0001444|broken|
|CCO:Square|Square|PATO:0000413|square|
|CCO:Straight|Straight|PATO:0002180|straight|
|CCO:Strength|Strength|PATO:0001230|strength|
|CCO:SurfaceTension|Surface Tension|PATO:0001461|surface tension|
|CCO:Telescope|Telescope|PATO:0002313|telescopic|
|CCO:Temperature|Temperature|PATO:0000146|temperature|
|CCO:Texture|Texture|PATO:0000150|texture|
|CCO:Thickness|Thickness|PATO:0000591|increased thickness|
|CCO:Thin|Thin|PATO:0000592|decreased thickness|
|CCO:Translucent|Translucent|PATO:0001354|translucent|
|CCO:Transparent|Transparent|PATO:0000964|transparent|
|CCO:Triangular|Triangular|PATO:0001875|triangular|
|CCO:Tripod|Tripod|PATO:0002428|tripodal|
|CCO:Velocity|Velocity|PATO:0002242|velocity|
|CCO:Vermilion|Vermilion|PATO:0001302|vermilion|
|CCO:Violet|Violet|PATO:0001424|violet|
|CCO:Wavelength|Wavelength|PATO:0001242|wavelength|
|CCO:Wavy|Wavy|PATO:0000967|undulate|
|CCO:Weight|Weight|PATO:0000128|weight|
|CCO:Wetness|Wetness|PATO:0001822|wetness|
|CCO:White|White|PATO:0000323|white|
|CCO:Wide|Wide|PATO:0000600|increased width|
|CCO:Width|Width|PATO:0000921|width|
|CCO:Yellow|Yellow|PATO:0000324|yellow|
|CCO:definition|definition|IAO:0000115|definition|


## ENVO

|ENVO|ENVO label|CCO|CCO label|
|---|---|---|---|
|ENVO:00000000|geographic feature|CCO:GeographicFeature|Geographic Feature|
|ENVO:00000005|first-order administrative region|CCO:FirstOrderAdministrativeRegion|First-Order Administrative Region|
|ENVO:00000006|second-order administrative region|CCO:SecondOrderAdministrativeRegion|Second-Order Administrative Region|
|ENVO:00000007|third-order administrative region|CCO:ThirdOrderAdministrativeRegion|Third-Order Administrative Region|
|ENVO:00000008|fourth-order administrative region|CCO:FourthOrderAdministrativeRegion|Fourth-Order Administrative Region|
|ENVO:00000009|national geopolitical entity|CCO:Country|Country|
|ENVO:00000011|garden|CCO:Park|Park|
|ENVO:00000012|hydrographic feature|CCO:HydrographicFeature|Hydrographic Feature|
|ENVO:00000014|canal|CCO:Canal|Canal|
|ENVO:00000025|reservoir|CCO:Reservoir|Reservoir|
|ENVO:00000029|watercourse|CCO:Branched|Branched|
|ENVO:00000040|waterfall|CCO:Force|Force|
|ENVO:00000062|populated place|CCO:PopulatedPlace|Populated place|
|ENVO:00000064|road|CCO:Highway|Highway|
|ENVO:00000065|railway|CCO:Railway|Railway|
|ENVO:00000066|man-made tunnel|CCO:Tunnel|Tunnel|
|ENVO:00000070|construction|CCO:ConstructedFeature|Constructed Feature|
|ENVO:00000074|dam|CCO:Dam|Dam|
|ENVO:00000075|bridge|CCO:Bridge|Bridge|
|ENVO:00000076|mine|CCO:Mine|Mine|
|ENVO:00000077|agricultural feature|CCO:AgriculturalFacility|Agricultural Facility|
|ENVO:00000078|farm|CCO:Farm|Farm|
|ENVO:00000087|cliff|CCO:Scar|Scar|
|ENVO:00000101|delta|CCO:Fan|Fan|
|ENVO:00000104|undersea feature|CCO:Flat|Flat|
|ENVO:00000122|track|CCO:Trail|Trail|
|ENVO:00000158|causeway|CCO:Road|Road|
|ENVO:00000191|solid astronomical body part|CCO:PhysiographicFeature|Physiographic Feature|
|ENVO:00000533|landfill|CCO:Landfill|Landfill|
|ENVO:00000563|pier|CCO:Pier|Pier|
|ENVO:00000856|city|CCO:City|City|
|ENVO:00002000|slope|CCO:Sloped|Sloped|
|ENVO:00002173|hospital|CCO:Hospital|Hospital|
|ENVO:00002221|shop|CCO:Shop|Shop|
|ENVO:00002271|nuclear power plant|CCO:NuclearPowerPlant|Nuclear Power Plant|
|ENVO:00002297|environmental feature|CCO:EnvironmentalFeature|Environmental Feature|
|ENVO:00002874|air conditioning unit|CCO:AirConditioningUnit|Air Conditioning Unit|
|ENVO:00003963|warehouse|CCO:Warehouse|Warehouse|
|ENVO:01000162|organ|CCO:Organization|Organization|
|ENVO:01000417|house|CCO:House|House|
|ENVO:01000460|asphalt|CCO:Pitch|Pitch|
|ENVO:01000536|factory|CCO:Factory|Factory|
|ENVO:01000577|radio device|CCO:RadioReceiver|Radio Receiver|
|ENVO:01000580|telephone|CCO:Telephone|Telephone|
|ENVO:01000582|landline telephone|CCO:LandlineTelephone|Landline Telephone|
|ENVO:01000602|truck|CCO:Truck|Truck|
|ENVO:01000604|vehicle|CCO:Vehicle|Vehicle|
|ENVO:01000610|motor|CCO:Engine|Engine|
|ENVO:01000611|bus|CCO:Bus|Bus|
|ENVO:01000613|watercraft|CCO:Watercraft|Watercraft|
|ENVO:01000614|bicycle|CCO:Bicycle|Bicycle|
|ENVO:01000615|motorcycle|CCO:Motorcycle|Motorcycle|
|ENVO:01000733|stock|CCO:Stock|Stock|
|ENVO:01000773|village|CCO:Village|Village|
|ENVO:01000779|public infrastructure|CCO:Infrastructure|Infrastructure|
|ENVO:01000984|grocery store|CCO:GroceryStore|Grocery Store|
|ENVO:01001221|office building|CCO:OfficeBuilding|Office Building|
|ENVO:01001488|aircraft|CCO:Aircraft|Aircraft|
|ENVO:2000037|fossil fuel power plant|CCO:FossilFuelPowerPlant|Fossil Fuel Power Plant|

**Comment by robsyc** [comment]
Hello, I only recently found out about this project and I was wondering what the relation is between this and [COB](https://obofoundry.org/ontology/cob.html)

The COB is still in development but I think the idea is also to provide a mid-level ontology for other domain-ontologies in OBO to find common ground with. Are there COO contributors who also work on COB or is this completely independent?

I think it could be a good place for OBO and CCO to find consensus.
